tsconfig.build.json

```diff
@@ -0,0 +1,95 @@
+{
+  "compilerOptions": {
+    "target": "ESNext",
+
+    "module": "ESNext",
+
+    "moduleResolution": "node",
+
+    "lib": ["esnext.array", "esnext", "es2017", "dom"],
+
+    "declaration": true,
+
+    "declarationMap": true,
+
+    "sourceMap": true,
+
+    "inlineSources": true,
+
+    "strict": true,
+
+    "noUnusedLocals": true,
+
+    "noUnusedParameters": true,
+
+    "noImplicitReturns": true,
+
+    "noFallthroughCasesInSwitch": true,
+
+    "noImplicitAny": true,
+
+    "noImplicitThis": true,
+
+    "experimentalDecorators": true,
+
+    "resolveJsonModule": true,
+
+    "allowSyntheticDefaultImports": true,
+  },
+  "include": ["src/**/*.ts", "types/*.ts"],
+  "exclude": ["node_modules", "dist"]
+}
+
+
+// {
```

Comment:
it is not valid to add `//` comments in a JSON file

Reply:
Done ✅